### ISSUE TITLE
Question about matrix in CalibrationPointsSquaresFunction.cpp

### ISSUE THREAD

**Issue by xiaojieguo** [coconetlero/US_Probe_Calibration#1]
Hi, I‘m reading your code now. And I want to use levenberg marquardt algorithm. 
I'm not sure about some matrixes in this file _CalibrationPointsSquaresFunction.cpp_
I think the matrixes are as follows, is it right ?

```
vnl_matrix<double> cTt(4, 4);       //transform matrix from US image to US probe
vnl_matrix<double> tTr(4, 4);        // transform matrix from  US probe to reference 
vnl_matrix<double> rTp(4, 4);       // transform matrix from  reference to phantom
vnl_matrix<double> xP(4,1);          //  point in the phantom
```

**Comment by coconetlero** [comment]
Hello. 

You can get more information in this paper:
Prager 1998 - Rapid Calibration for 3D Freehand Ultrasound

I used the same notation as referred in that paper.

**Comment by xiaojieguo** [comment]
Thanks! Could the software connect to NDI Polaris ?

**Comment by coconetlero** [comment]
Yes you can, but I don't have the implementation of the connection with the NDI polaris, but is relativeley easy if you use IGSTK. There are a plenty of examples of how to do that.
